airflow/config_templates/default_celery.py

```diff
@@ -67,14 +68,14 @@ def _broker_supports_visibility_timeout(url):
 
 try:
     if celery_ssl_active:
-        if broker_url and "amqp://" in broker_url:
+        if broker_url and re.search("amqp://", broker_url):
```

Comment:
Please don’t use `re.search` for this, it’s a lot slower. I can _maybe_ see the point using it for more complex matches below, but this one definitely should not.